src/components/structures/UserSettings.js

```diff
@@ -46,7 +46,7 @@ const gHVersionLabel = function(repo, token='') {
     } else {
         url = `https://github.com/${repo}/commit/${token.split('-')[0]}`;
     }
-    return <a href={url}>{token}</a>;
+    return <a target="_blank" href={url}>{token}</a>;
```

Comment:
We need to be very careful with `target="_blank"` as per https://mathiasbynens.github.io/rel-noopener/ (tl;dr, add `rel="noopener"`)